.yamato/metadata.metafile

```diff
@@ -38,10 +38,19 @@ all_testprojects:
     - name: HDRP
       path: Projects/HDRP
       editors: [2022.2, trunk]
+    - name: URPInputSystem
+      path: Projects/URPInputSystem
+      editors: [2022.2, trunk]
+    - name: StandaloneInputSystem
+      path: Projects/StandaloneInputSystem
+      editors: [2022.2, trunk]
+    - name: HDRPInputSystem
+      path: Projects/HDRPInputSystem
+      editors: [2022.2, trunk]
 
 all_configurations:
   ci:
-    test_names: [isolation_tests]
+    test_names: [isolation_tests, promotion_tests]
```

Comment:
I think @guillaumelevass removed promotion tests for a reason. I might be mistaken :)

Reply:
I checked git history and that's not the case. @guillaumelevass did not remove this. He add promotion tests to the nightly only, so I remembered wrong. :)

Why did you add it?

Reply:
Fixed. Was just a test.